packages/@aws-cdk/custom-resource-handlers/lib/aws-stepfunctions-tasks/cross-region-aws-sdk-handler/index.ts

```diff
@@ -0,0 +1,79 @@
+/* eslint-disable @typescript-eslint/no-require-imports */
+/* eslint-disable no-console */
+
+type Event = {
+  /**
+   * AWS Region name to call the service
+   */
+  region: string;
+
+  /**
+   * AWS service name (must be in AWS SDK for JS v3 style)
+   */
+  service: string;
+
+  /**
+   * AWS API endpoint
+   */
+  endpoint: string;
+
+  /**
+   * API Action name (must be in camelCase or PascalCase)
+   */
+  action: string;
+
+  /**
+   * request parameters to call the API (must be in AWS SDK for JS v3 style)
+   */
+  parameters?: any;
```

Comment:
I couldn't find a common interface or union for all `InputTypes` across services, but this should be a safe bet?

```suggestion
  parameters?: Record<string, any>;
```

Reply:
Hi @nmussy, thanks. Given that we cannot validate it in compile time nor synth time, I think it will not add much value if we narrow its type definition. It will result in a runtime error anyway.